docs/source/en/peft.md

```diff
@@ -208,6 +208,26 @@ model.save_pretrained(save_dir)
 model = AutoModelForCausalLM.from_pretrained(save_dir)
 ```
 
+## Add additional trainable layers to a PEFT adapter
```

Comment:
Do we typically advertise PEFT features in the docs if they're only on dev? I'm OK with it in principle, but I'm expecting a lot of issues with people complaining that this doesn't work

Reply:
We plan to do a patch release tonight for it cc @pacman100